integration-tests/main/src/main/resources/application.properties

```diff
@@ -37,3 +37,5 @@ configproperties.message=Hello
 configproperties.other=1
 
 quarkus.swagger-ui.always-include=true
+
+quarkus.native.resources.includes = test-resources/.*\\\\.txt
```

Comment:
Oh my, this is very ugly, especially since it will be very common :/

@dmlloyd any idea?

Reply:
It is hard to do anything as long as we keep using regular expressions and MP config list converter.

One way out could be to use globs instead of REs. Globs are a bit less expressive than REs, but much easier to use for the 99% of common cases. With globs, the above would be `quarkus.native.resources.includes = test-resources/**.txt`.

Using a custom list converter with some different escaping mechanism could eliminate two of the four backslashes - not sure it is worth the effort.

Reply:
Maybe globs are better, what do you think @gsmet ?